thalo-eventstoredb/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+pub use event_store::{ESDBEventStore, ESDBEventPayload};
```

Comment:
I've been trying to be good with documentation, so it would be nice if we could have `#![deny(missing_docs)]` and the top of this file and have some basic documentation written.